extensions/ipynb/package.json

```diff
@@ -51,6 +51,16 @@
 				"priority": "default"
 			}
 		],
+    "notebookRenderer": [
+      {
+        "id": "vscode.markdown-it-cell-attachment-renderer-extension",
```

Comment:
Nit, this is the id for the renderer so maybe call it `vscode.markdown-it-cell-attachment-renderer`